pkgs/tools/wayland/wl-clipboard/default.nix

```
@@ -21,18 +23,24 @@ stdenv.mkDerivation rec {
   };
 
   strictDeps = true;
-  nativeBuildInputs = [ meson ninja pkg-config wayland-scanner ];
+  nativeBuildInputs = [ meson ninja pkg-config wayland-scanner makeWrapper ];
   buildInputs = [ wayland wayland-protocols ];
 
   mesonFlags = [
     "-Dfishcompletiondir=share/fish/vendor_completions.d"
   ];
 
+  # Fix for https://github.com/NixOS/nixpkgs/issues/251261
+  postInstall = ''
```

```suggestion
  postInstall = 'lib.optionalString (!xdg-utils.meta.broken) '
```

xdg-utils doesn't cross-compile

---

fixed in https://github.com/NixOS/nixpkgs/pull/251474/commits/d4ef31fc1464d1f806aaa329b8c000e586867d9c